hudi-spark-datasource/hudi-spark3.2plus-common/src/main/scala/org/apache/spark/sql/hudi/analysis/HoodieSpark32PlusAnalysis.scala

```diff
@@ -104,6 +104,21 @@ case class HoodieSpark32PlusResolveReferences(spark: SparkSession) extends Rule[
           catalogTable.location.toString))
         LogicalRelation(relation, catalogTable)
       }
+    case HoodieQueryTimeline(args) =>
```

Comment:
please also check other `Analysis` classes for different Spark versions. May have to do this in those classes too.

Reply:
Could not find TVF support in other places.